jupyter-docker/Dockerfile

```diff
@@ -126,8 +126,7 @@ RUN apt-get install -yq --no-install-recommends \
  && pip install biopython \
  && pip install bx-python \
  && pip install fastinterval \
- && pip install matplotlib-venn \
- && pip install "https://github.com/all-of-us/workbench/archive/pyclient-v0-1-rc3.zip#egg=aou_workbench_client&subdirectory=client/py"
```

Comment:
Note: the latest version of this library would be:

https://github.com/all-of-us/pyclient/archive/pyclient-v1-1-rc3.zip#egg=aou_workbench_client&subdirectory=py

But I think we want to start passing the version to use as a startup script, rather than having it hardcoded here... does that sound right @helgridly?

Reply:
Yeah that was my thinking -- not install it by default in Leo, but let you install it via a custom script.